### PR TITLE
CHAT-1066: Fix scrolling manually

### PR DESCRIPTION
**Pull request by hassenelkamel-exo**
- To apply the scroll to the last message, we hidden scroll in drawer in list message view and we activate scrolling in message list div (taskid=23697).
- i divide the css into 2 class:
1. class contentDrawer: define the style of the list of contact.
2. class contentDrawerOfList: define the style of list of message

### CODE REVIEW

File: application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue [hunk 1]
```diff
@@ -25,7 +25,7 @@
           <a v-show = "!showSearch" class="uiIconSearchLight" @click="showSearch = !showSearch"></a>
         </span>
       </div>
-      <div :class="showChatDrawer ? 'contentDrawer ' : '' " class="content">
+      <div :style="[!listOfContact ? {overflow: 'hidden'}:{}]" :class="showChatDrawer ? 'contentDrawer ' : '' " class="content">
```
Comment (ahamdi) [inline application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue:28]: Preferably, we manage styles suing just CSS classes. you can use v-bind:class with a clear syntax as you can check https://vuejs.org/v2/guide/class-and-style.html

Comment (hassenelkamel-exo) [reply application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue:28]: if you add :class we have two binding, and scrolling in drawer must be in 1 view(list of contacts), only we want enabled in the view of list of message

Comment (ahamdi) [reply application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue:28]: I meant could you combine both in one binding using classes

Comment (hassenelkamel-exo) [reply application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue:28]: we have tow different condition, we cannot combine;

Comment (hassenelkamel-exo) [reply application/src/main/webapp/vue-app/components/modal/ExoChatDrawer.vue]: ok done

**Pull request merged**